{"ip_str": "104.238.182.217", "errors": {"VPN_error": "Timeout exceeded.\n<pexpect.pty_spawn.spawn object at 0x7fce09f0c370>\ncommand: /usr/sbin/openvpn\nargs: ['/usr/sbin/openvpn', '/home/NetPlatform/configurations/bulletVPN-TW-Taipei-tai01-udp.ovpn']\nbuffer (last 100 chars): b\" C=US, O=Let's Encrypt, CN=R3\\r\\nTue Nov  1 07:51:35 2022 VERIFY OK: depth=0, CN=tai01.bulletvpn.com\\r\\n\"\nbefore (last 100 chars): b\" C=US, O=Let's Encrypt, CN=R3\\r\\nTue Nov  1 07:51:35 2022 VERIFY OK: depth=0, CN=tai01.bulletvpn.com\\r\\n\"\nafter: <class 'pexpect.exceptions.TIMEOUT'>\nmatch: None\nmatch_index: None\nexitstatus: None\nflag_eof: False\npid: 9\nchild_fd: 5\nclosed: False\ntimeout: 30\ndelimiter: <class 'pexpect.exceptions.EOF'>\nlogfile: None\nlogfile_read: None\nlogfile_send: None\nmaxread: 2000\nignorecase: False\nsearchwindowsize: None\ndelaybeforesend: 0.05\ndelayafterclose: 0.1\ndelayafterterminate: 0.1\nsearcher: searcher_re:\n    0: re.compile(b'Initialization Sequence Completed')"}}